### PR TITLE
Avoid n+1 queries while creating a product

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | - |
| License         | MIT |

Before:
screen shot 2017-02-28 at 10 25 11

After:
screen shot 2017-02-28 at 10 35 46

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/AttributeRepository.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sylius\Bundle\CoreBundle\Doctrine\ORM;
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/AttributeRepository.php]: Missing license block :(

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductOptionRepository.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sylius\Bundle\CoreBundle\Doctrine\ORM;
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductOptionRepository.php]: And here.

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 imports:
+    - { resource: "@SyliusCoreBundle/Resources/config/app/sylius/sylius_attribute.yml" }
```
Comment (tuka217) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml]: Should be under addressing import.

**Comment by pjedrzejewski** [comment]
Merged in #7643.

**Pull request closed without merging**